modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -836,7 +836,7 @@ private RaftGroupOptions groupOptionsForPartition(boolean isVolatileStorage, @Nu
         }
 
         raftGroupOptions.snapshotStorageFactory(snapshotFactory);
-
+        raftGroupOptions.maxClockSkew((int) clockService.maxClockSkewMillis());
```

Comment:
To avoid cast, because electionTimeoutMs in NodeOptions is int.
`nodeOptions.setElectionTimeoutMs(Math.max(nodeOptions.getElectionTimeoutMs(), groupOptions.maxClockSkew()));`